internal/utils/log.go

```diff
@@ -32,6 +32,17 @@ func SetLogger() {
 		w.TimeFormat = time.RFC3339
 	}))
 
+	// Create a logger that logs to ttyS0
+	// Open the serial port for writing
+	serial, err := os.OpenFile("/dev/ttyS0", os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
This should be removed as it doesn't work

Reply:
does it not? same thing for kmesg below?

Reply:
I could only test it a couple of times but I think at this point the  /dev dir is not there and if it is it's not working as expected so I rather leave it to implement it properly later.

Plus we still got to move to the new Kairos logger :)

Reply:
done